packages/cli/src/generator/templates/Component.tsx.ejs

```diff
@@ -25,26 +25,28 @@ export const <%= componentName %>: React.SFC<<%= componentName %>Props> = props
 <% if (graphqlTypeName) { -%>
 export const <%= relayContainerName %> = create<%= relayContainerType %>(
   <%= componentName %>,
-  graphql`
-    fragment <%= relayTypeName %> on <%= graphqlTypeName %><%- relayContainerType === "PaginationContainer" ? " @argumentDefinitions(count: { type: \"Int\", defaultValue: 10 }, cursor: { type: \"String\", defaultValue: \"\" })" : "" %> {
+  {
+    <%= relayPropName %>: graphql`
+      fragment <%= relayTypeName %> on <%= graphqlTypeName %><%- relayContainerType === "PaginationContainer" ? " @argumentDefinitions(count: { type: \"Int\", defaultValue: 10 }, cursor: { type: \"String\", defaultValue: \"\" })" : "" %> {
 <% if (relayContainerType === "RefetchContainer" || relayContainerType === "PaginationContainer") { -%>
-      # Most, but not all, types have this field. If needed, replace it with a
-      # different identifier field and be sure to adjust the query below
-      # accordingly. (Also be sure to remove this comment.)
-      __id
+        # Most, but not all, types have this field. If needed, replace it with a
+        # different identifier field and be sure to adjust the query below
+        # accordingly. (Also be sure to remove this comment.)
+        __id
 <% if (relayContainerType === "PaginationContainer") { -%>
-      <%= relayPaginationFieldName %>(first: $count, after: $cursor) @connection(key: "<%= componentName %>_<%= relayPaginationFieldName %>") {
-        pageInfo {
-          hasNextPage
-        }
-        edges {
-          node {
+        <%= relayPaginationFieldName %>(first: $count, after: $cursor) @connection(key: "<%= componentName %>_<%= relayPaginationFieldName %>") {
+          pageInfo {
+            hasNextPage
+          }
+          edges {
+            node {
+            }
           }
         }
-      }
 <% } -%>
-    }
-  `,
+      }
+    `,
+  },
```

Comment:
I can't figure out where (these template files are :scream:), but we're missing a closing brace on fragment containers (pagination containers are fine)

![image](https://user-images.githubusercontent.com/1242537/57861382-690ea500-77ee-11e9-89dc-5c4bbd30bf25.png)

Reply:
Oh wow, great catch!

Reply:
I guess the test should ideally try to parse the GraphQL documents to ensure they aren't broken in these ways

Reply:
great idea :+1:

Reply:
Pretty happy with this:

<img width="753" alt="Screenshot 2019-05-20 at 18 52 35" src="https://user-images.githubusercontent.com/2320/58039151-2c1a1980-7b32-11e9-8059-a7e90e5eb763.png">

Reply:
Yessss thank you :pray: